### PR TITLE
ENH: Readme

### PR DESCRIPTION
**Pull request by AntoineTheb**
Add readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+## Getting started
+
+You will need to have scilpy scripts loaded as part of your environment. To run the base version of the pipeline on the training data, you can run:
+
+```bash
+$ cd code
+$ bash -x DiSCo_connectivity_pipeline.sh ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_DWI_RicianNoise-snr30.nii.gz ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_DWI_RicianNoise-snr30.bval ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_DWI_RicianNoise-snr30.bvec ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_ROIs.nii.gz ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_Connectivity_Matrix_Cross-Sectional_Area.txt pft training
+```
+
+which should net you an r coefficient of around ~0.80.
```
Comment (CHrlS98) [inline README.md:39]: Typo? "get", right?

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+- Step 12 : Compute binary correlation
+- Step 13 : Compute confusion matrix
+
+## Getting started
+
+You will need to have scilpy scripts loaded as part of your environment. To run the base version of the pipeline on the training data, you can run:
+
+```bash
+$ cd code
+$ bash -x DiSCo_connectivity_pipeline.sh ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_DWI_RicianNoise-snr30.nii.gz ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_DWI_RicianNoise-snr30.bval ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_DWI_RicianNoise-snr30.bvec ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_ROIs.nii.gz ~/braindata/databases/DiSCo/sub-DiSCo1/sub-DiSCo1_Connectivity_Matrix_Cross-Sectional_Area.txt pft training
```
Comment (CHrlS98) [inline README.md:36]: What is `sub-DiSCo` again? Should it be discussed in section `Data`?

**Review by CHrlS98 — COMMENTED** [review]
Very clear !
Some minor comments.

File: README.md (hunk 1 above)
Comment (AntoineTheb) [reply README.md:39]: "net" as in the verb "to net" https://www.merriam-webster.com/dictionary/net :)

File: README.md (hunk 2 above)
Comment (AntoineTheb) [reply README.md:36]: It's just BIDS ( https://bids.neuroimaging.io/) nomenclature, short for subjet-DiSCo[1-3] but I'll update the README to be clearer.

Comment (AntoineTheb) [reply README.md:36]: Donesies in 4a12492

**Review by CHrlS98 — APPROVED** [review]
Everything is very clear !

File: README.md (hunk 1 above)
Comment (CHrlS98) [reply README.md:39]: oops my bad!

**Pull request merged**